services/autorust/azure-autorust/src/main.rs

```diff
@@ -14,52 +14,63 @@ struct Args {
     /// Generate the publish GitHub workflows
     #[clap(long)]
     publish: bool,
+
+    /// Specify specific package to generate
+    #[clap(long = "package", short = 'p')]
+    packages: Vec<String>,
+}
+impl Args {
+    pub fn packages(&self) -> Vec<&str> {
+        self.packages.iter().map(String::as_str).collect()
```

Comment:
Just a thought: This seems somewhat wasteful - maybe returning `impl Iterator<Item = &str>` would work better?